src/docs/releases/2.0.0.md

```diff
@@ -289,40 +289,40 @@ Here are the updated signatures:
 
 These adjustments ensure compatibility and adherence to the latest conventions within the `DisplayDriver` class.
 
-Additionally, we've enhanced display performance by streamlining the display drivers. To achieve this, the following methods have been marked as obsolete:
+Additionally, we've enhanced display performance by optimizing the display drivers. The following methods have been marked as obsolete:
 
 - `IDisplayResult Display(TModel model)`
 - `IDisplayResult Edit(TModel model)`
 
-and removed the following methods:
+the following methods have been removed, with updated alternatives provided:
 
-- `Task<IDisplayResult> DisplayAsync(TModel model, IUpdateModel updater)`
-- `IDisplayResult Display(TModel model, IUpdateModel updater)`
-- `Task<IDisplayResult> UpdateAsync(TModel model, IUpdateModel updater)`
-- `Task<IDisplayResult> EditAsync(TModel model, IUpdateModel updater)`
-- `IDisplayResult Edit(TModel model, IUpdateModel updater)`
+- `Task<IDisplayResult> DisplayAsync(TModel model, IUpdateModel updater)` → Use `DisplayAsync(TModel model, BuildDisplayContext context)` where the `Updater` property is available via `context`.
+- `IDisplayResult Display(TModel model, IUpdateModel updater)` → Use `Display(TModel model, BuildDisplayContext context)` where the `Updater` property is available via `context`.
+- `Task<IDisplayResult> UpdateAsync(TModel model, IUpdateModel updater)` → Use `UpdateAsync(TModel model, UpdateEditorContext context)` where the `Updater` property is available via `context`.
+- `Task<IDisplayResult> EditAsync(TModel model, IUpdateModel updater)` → Use `EditAsync(TModel model, BuildEditorContext context)` where the `Updater` property is available via `context`.
+- `IDisplayResult Edit(TModel model, IUpdateModel updater)` → Use `Edit(TModel model, BuildEditorContext context)` where the `Updater` property is available via `context`.
 
-At the same time, we've introduced the following static methods in the `DisplayDriverBase`:
+In addition, we've introduced new static methods in the `DisplayDriverBase` class:
 
 - `Task<IDisplayResult> CombineAsync(params IDisplayResult[] results)`
 - `Task<IDisplayResult> CombineAsync(IEnumerable<IDisplayResult> results)`
 
-Similar refactoring was done to `SectionDisplayDriver` class. The following methods have been marked as obsolete:
+The same refactoring applies to the `SectionDisplayDriver` class. The following methods are now obsolete:
 
 - `Display(TSection section, BuildDisplayContext context)`
 - `Display(TSection section)`
 - `Edit(TSection section, BuildEditorContext context)`
 - `UpdateAsync(TSection section, UpdateEditorContext context)`
 - `UpdateAsync(TSection section, IUpdateModel updater, string groupId)`
 
-and removed the following methods:
+And the following methods have been removed, with updated alternatives:
```

Comment:
```suggestion
The following methods have been removed, with updated alternatives:
```

Reply:
why do you think this is better? its a continuation of the previous part.

Reply:
Ok leave it